.golangci.yml

```diff
@@ -10,9 +10,7 @@ run:
     - pql/pql.peg.go
 linters:
   enable:
-    - govet
-    - gofmt
-    - staticcheck
+    - gosimple
```

Comment:
Is this change intentional? It seems surprising to me because I was pretty sure we did in fact want those three, especially staticcheck.